nucypher/characters/control/specifications/fields/misc.py

```diff
@@ -0,0 +1,34 @@
+import click
+from marshmallow import fields
+from nucypher.characters.control.specifications.fields.base import BaseField
+from nucypher.cli import types
+
+
+class String(BaseField, fields.String):
+    pass
+
+
+class List(BaseField, fields.List):
+    pass
+
+
+class Integer(BaseField, fields.Integer):
```

Comment:
I'd create a PositiveInteger subclass with a `_validate()`, as most (all?) integer fields are actually intended to be positive.

Reply:
already done.  next push!